### ISSUE TITLE
API for end of game state / winner

### ISSUE THREAD

**Issue by d12** [sendwithus/battlesnake-server#38]
After the game stops querying the snake for moves and the game is over, I wasn't able to find a way to query for the winners of the game programatically. Is this something that exists, or can an API be exposed for this?

**Comment by exzizt** [comment]
I think this is a cool idea. If there was a hook for /win /tie /lose that sent the game id as well as the final state it would allow for good data analysis.

**Comment by d12** [comment]
I ended up adding a super jank addition to the server, it just sends off a request during the set_winners callback. It could be configured to send to a snakes URL, or to each snakes URL:

```diff
@@ -102,6 +106,15 @@ defmodule Bs.GameState do
     winners = if length(winners) != 0, do: winners, else: who_died_last(state)
     winners = MapSet.new(winners)
     put_in(state.winners, winners)
+
+    data = Poison.encode!(%{"winners": winners, "game_id": state.world.game_id})
+
+    :timer.tc(@http, :post!, [
+      "http://localhost:8000",
+      data,
+      ["Content-Type": "application/json"],
+      [recv_timeout: 1000]
+    ])
```

**Comment by brandon-swu** [comment]
Thanks for this @d12, well take a look at getting this in

**Comment by daniel-swu** [comment]
Fixed by pr #46
